### DISCUSSION TITLE
Cannot change visibility of repository

### DISCUSSION THREAD

**Discussion by Bohun9** [community/community#46870] [New to GitHub] [answered]
I noticed that I can change repositories from private to public, but not from public to private. I get a screen where only clickable object is exit...
![image](https://user-images.githubusercontent.com/79287762/217856459-bee0fabb-0513-45bf-be0e-247489cc6d64.png)

**Comment by SilasRodrigues19** [comment]
Hey @Bohun9 

Are you accessing through the GitHub app? Because you can do this through the website, regardless of whether you access it through desktop or mobile

**Reply by Bohun9** [reply]
Hey @SilasRodrigues19 

I'm using github through website. I'm following steps that are written in documentation: repository settings -> danger zone -> change repository visibility.

**Comment by SilasRodrigues19** [comment]
The option appears normal to me.

![image](https://user-images.githubusercontent.com/47532703/217867057-096ca65b-41cc-4bf8-86d5-b8b0b09b3ed3.png)
![image](https://user-images.githubusercontent.com/47532703/217867179-ecca14a5-16b7-4600-a55f-0c1191db95de.png)

**Comment by Bohun9** [comment]
The screenshot was the screen I got after clicking "change to private" and "I want to make this repository private".

I don't know why, but now it works and it shows additional box at the bottom.

![image](https://user-images.githubusercontent.com/79287762/217869379-10c981e3-6e39-4ae5-86c9-b555f6128b3a.png)

Anyway, thanks for help :)

**Reply by radar-blip** [reply]
Thats just a warning so you understand the consequences. 👍🏻

**Comment by Bohun9** [comment, marked as answer]
Problem was that I was zoomed in and the button was cut off.

**Reply by queenofcorgis** [reply]
Love when it is a simple fix - thanks for coming back to share and for marking the answer @Bohun9 ⭐
